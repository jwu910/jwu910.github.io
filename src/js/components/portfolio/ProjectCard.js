import React from 'react';

class ProjectCard extends React.Component {
  render() {
    const { title } = this.props;

    //@todo Render create link to component based on title

    return (
      <div className='project-card-wrapper'>
        <div className="title">{ title }</div>
      </div>
    );
  }
}

export default ProjectCard;