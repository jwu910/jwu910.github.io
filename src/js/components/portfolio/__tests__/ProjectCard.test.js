import React from 'react';
import renderer from 'react-test-renderer';

import ProjectCard from '../ProjectCard';

describe('Snapshots', () => {
  it('Renders correctly', () => {
    const component = renderer
      .create(
        <ProjectCard />
      )
      .toJSON();

    expect(component).toMatchSnapshot();
  });
});