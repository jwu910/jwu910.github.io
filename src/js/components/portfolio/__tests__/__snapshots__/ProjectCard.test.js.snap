// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshots Renders correctly 1`] = `
<div
  className="project-card-container"
>
  <a
    className="githubLink"
    href="https://github.comundefined"
  >
    <div
      className="card-wrapper"
    >
      <div
        className="card-box"
      >
        <div
          className="card-title"
        >
          <span
            className="title"
          />
        </div>
        <div
          className="card-content"
        />
      </div>
    </div>
  </a>
</div>
`;
